src/backend/distributed/commands/citus_add_local_table_to_metadata.c

```diff
@@ -1019,6 +1029,30 @@ GetDropTriggerCommand(Oid relationId, char *triggerName)
 }
 
 
+/*
+ * DropViewsOnTable drops the views that depend on the given relation.
+ */
+static void
+DropViewsOnTable(Oid relationId)
+{
+	List *views = GetDependingViews(relationId);
+
+	Oid viewId = InvalidOid;
+	foreach_oid(viewId, views)
+	{
+		char *viewName = get_rel_name(viewId);
+		char *schemaName = get_namespace_name(get_rel_namespace(viewId));
+		char *qualifiedViewName = quote_qualified_identifier(schemaName, viewName);
```

Comment:
Could you please add test to check whether we drop view properly if view has different schema than table?